src/entities/amm/amm.ts

```diff
@@ -688,8 +697,8 @@ export class AMM {
       : fixedTokenDeltaUnbalanced.mul(BigNumber.from(1000)).div(availableNotional).toNumber() /
         1000;
 
-    const swapGasUnits = await peripheryContract.estimateGas.swap(swapPeripheryParams);
-    const gasFeeETH = await convertGasUnitsToETH(this.provider, swapGasUnits.toNumber());
+    // const swapGasUnits = await peripheryContract.estimateGas.swap(swapPeripheryParams);
```

Comment:
why do we comment this?

Reply:
It fails if the counterfactual swap fails, as we discussed last week.